### ISSUE TITLE
No active editor

### ISSUE THREAD

**Issue by tony1157** [emmetio/livestyle-sublime-old#20]
after installed all plug-ins and tried in chrome it shows"No active editor"
rebooted computer again and again, trued to reinstall ST2 and chrome for many times, it not work

my system is : win 7 64bit, and chrome is 28.0.1500.95 m

**Comment by sergeche** [comment]
Can you post your ST console log here?

**Comment by tony1157** [comment]
here is the console 

[code]
startup, version: 2221 windows x64 channel: stable
executable: /C/Program Files/Sublime Text 2/sublime_text.exe
working dir: /C/Windows/system32
packages path: /C/Users/Tonyleung/AppData/Roaming/Sublime Text 2/Packages
settings path: /C/Users/Tonyleung/AppData/Roaming/Sublime Text 2/Settings
PackageSetup not required
Py_GetProgramName(): sublime_text.exe
Py_GetExecPrefix(): C:\Program Files\Sublime Text 2
Py_GetProgramFullPath(): C:\Program Files\Sublime Text 2\sublime_text.exe
Py_GetPath(): C:\Program Files\Sublime Text 2\python26.zip;C:\Program Files\Sublime Text 2\DLLs;C:\Program Files\Sublime Text 2\lib;C:\Program Files\Sublime Text 2\lib\plat-win;C:\Program Files\Sublime Text 2\lib\lib-tk;C:\Program Files\Sublime Text 2
Py_GetPythonHome(): C:\Program Files\Sublime Text 2
catalogue loaded
found 6 files for base name Default.sublime-keymap
found 1 files for base name Default.sublime-mousemap
found 6 files for base name Main.sublime-menu
Reloading plugin C:\Users\Tonyleung\AppData\Roaming\Sublime Text 2\Packages\Alignment\Alignment.py
Reloading plugin C:\Users\Tonyleung\AppData\Roaming\Sublime Text 2\Packages\CSS\css_completions.py
Reloading plugin C:\Users\Tonyleung\AppData\Roaming\Sublime Text 2\Packages\CSSFormat\Cssformat.py
Reloading plugin C:\Users\Tonyleung\AppData\Roaming\Sublime Text 2\Packages\Default\comment.py
Reloading plugin C:\Users\Tonyleung\AppData\Roaming\Sublime Text 2\Packages\Default\copy_path.py
Reloading plugin C:\Users\Tonyleung\AppData\Roaming\Sublime Text 2\Packages\Default\delete_word.py
Reloading plugin C:\Users\Tonyleung\AppData\Roaming\Sublime Text 2\Packages\Default\detect_indentation.py
Reloading plugin C:\Users\Tonyleung\AppData\Roaming\Sublime Text 2\Packages\Default\duplicate_line.py
Reloading plugin C:\Users\Tonyleung\AppData\Roaming\Sublime Text 2\Packages\Default\echo.py
Reloading plugin C:\Users\Tonyleung\AppData\Roaming\Sublime Text 2\Packages\Default\exec.py
Reloading plugin C:\Users\Tonyleung\AppData\Roaming\Sublime Text 2\Packages\Default\fold.py
Reloading plugin C:\Users\Tonyleung\AppData\Roaming\Sublime Text 2\Packages\Default\font.py
Reloading plugin C:\Users\Tonyleung\AppData\Roaming\Sublime Text 2\Packages\Default\goto_line.py
Reloading plugin C:\Users\Tonyleung\AppData\Roaming\Sublime Text 2\Packages\Default\indentation.py
Reloading plugin C:\Users\Tonyleung\AppData\Roaming\Sublime Text 2\Packages\Default\kill_ring.py
Reloading plugin C:\Users\Tonyleung\AppData\Roaming\Sublime Text 2\Packages\Default\mark.py
Reloading plugin C:\Users\Tonyleung\AppData\Roaming\Sublime Text 2\Packages\Default\new_templates.py
Reloading plugin C:\Users\Tonyleung\AppData\Roaming\Sublime Text 2\Packages\Default\open_file_settings.py
Reloading plugin C:\Users\Tonyleung\AppData\Roaming\Sublime Text 2\Packages\Default\open_in_browser.py
Reloading plugin C:\Users\Tonyleung\AppData\Roaming\Sublime Text 2\Packages\Default\paragraph.py
Reloading plugin C:\Users\Tonyleung\AppData\Roaming\Sublime Text 2\Packages\Default\save_on_focus_lost.py
Reloading plugin C:\Users\Tonyleung\AppData\Roaming\Sublime Text 2\Packages\Default\scroll.py
Reloading plugin C:\Users\Tonyleung\AppData\Roaming\Sublime Text 2\Packages\Default\set_unsaved_view_name.py
Reloading plugin C:\Users\Tonyleung\AppData\Roaming\Sublime Text 2\Packages\Default\side_bar.py
Reloading plugin C:\Users\Tonyleung\AppData\Roaming\Sublime Text 2\Packages\Default\sort.py
Reloading plugin C:\Users\Tonyleung\AppData\Roaming\Sublime Text 2\Packages\Default\swap_line.py
Reloading plugin C:\Users\Tonyleung\AppData\Roaming\Sublime Text 2\Packages\Default\switch_file.py
Reloading plugin C:\Users\Tonyleung\AppData\Roaming\Sublime Text 2\Packages\Default\transform.py
Reloading plugin C:\Users\Tonyleung\AppData\Roaming\Sublime Text 2\Packages\Default\transpose.py
Reloading plugin C:\Users\Tonyleung\AppData\Roaming\Sublime Text 2\Packages\Default\trim_trailing_white_space.py
Reloading plugin C:\Users\Tonyleung\AppData\Roaming\Sublime Text 2\Packages\Diff\diff.py
Reloading plugin C:\Users\Tonyleung\AppData\Roaming\Sublime Text 2\Packages\Emmet\emmet-plugin.py
Emmet: No need to update PyV8
Reloading plugin C:\Users\Tonyleung\AppData\Roaming\Sublime Text 2\Packages\HTML\encode_html_entities.py
Reloading plugin C:\Users\Tonyleung\AppData\Roaming\Sublime Text 2\Packages\HTML\html_completions.py
Reloading plugin C:\Users\Tonyleung\AppData\Roaming\Sublime Text 2\Packages\LiveStyle\livestyle.py
Reloading plugin C:\Users\Tonyleung\AppData\Roaming\Sublime Text 2\Packages\Package Control\Package Control.py
plugin init time: 0.330958
loading bindings
loading pointer bindings
found 1 files for base name Default.sublime-theme
theme loaded
app ready
pre session restore time: 0.469747
wrote startup cache, added files: 2 orphaned files: 2 total files: 106 cache hits: 104
startup time: 0.555146 (package setup was not run)
loaded 1055 snippets
Package Control: Skipping automatic upgrade, last run at 2013-08-08 16:25:29, next run at 2013-08-08 17:25:29 or after
[/code]

**Comment by tony1157** [comment]
and got a socket error on checking background.html
Socket error: 
Event {clipboardData: undefined, cancelBubble: false, returnValue: true, srcElement: WebSocket, defaultPrevented: false…}
 background.js:18
Socket error: 
Event {clipboardData: undefined, cancelBubble: false, returnValue: true, srcElement: WebSocket, defaultPrevented: false…}

**Comment by sergeche** [comment]
OK, you got the same issue as some other users has. I’m trying to figure out what happening

**Comment by tony1157** [comment]
thanks for comment, hope it will soon be working :)

**Comment by turdi** [comment]
Same problem here... Waiting for solution! 

In general! Awesome work done!

**Comment by tony1157** [comment]
Hi guys, it seems that i've solved this problem.
My computer is in a local area network, and i set the LAN to a proxy getway, so the local address(127.0.0.1) has been occupied.
So, avoid using the proxy gateway would solve this problem.
Hope it help :)

**Comment by sergeche** [comment]
Oh, looks like @tony1157 saved the day :) I’ll try to figure out how to get access to local machine even with proxy enabled

**Comment by turdi** [comment]
Sorry for a stupid question... but I have everything set to automatic, so what should I do to solve this problem? 

I'm not connected to LAN, because I'm using WLAN. But I use XAMP. So If I understand it right it could cause this problem, only that I have no idea how to fix it...

**Comment by tony1157** [comment]
go chrome://extensions/ and find Livestyle 
open background.html and check the console log, it will help to solve

**Comment by sergeche** [comment]
@turdi do you see the same error as @tony1157 described in second comment? Or you just see `No active connection` message in LiveStyle panel? Check out ST console for errors

**Comment by turdi** [comment]
I feel realy stupid... How do I access to ST console? Went through menus and didn't find nothing to see errors.

**Comment by sergeche** [comment]
View > Show Console

**Comment by turdi** [comment]
DPI scale: 1
startup, version: 3047 windows x64 channel: stable
executable: /C/Program Files/Sublime Text 3/sublime_text.exe
working dir: /D/Work/freelancer/es tu ganga/html/css
packages path: /C/Users/Turdi Designs/AppData/Roaming/Sublime Text 3/Packages
state path: /C/Users/Turdi Designs/AppData/Roaming/Sublime Text 3/Local
hardware concurrency: 8
zip path: /C/Program Files/Sublime Text 3/Packages
zip path: /C/Users/Turdi Designs/AppData/Roaming/Sublime Text 3/Installed Packages
found 3 files for base name Default.sublime-keymap
found 1 files for base name Default.sublime-mousemap
found 2 files for base name Main.sublime-menu
loading bindings
loading pointer bindings
found 1 files for base name Default.sublime-theme
theme loaded
app ready
startup cache, total files: 148 cache hits: 148
pre session restore time: 0.047791
startup time: 0.079791
first paint time: 0.080791
launching: /C/Program Files/Sublime Text 3/plugin_host.exe
reloading plugin Default.block
reloading plugin Default.comment
reloading plugin Default.copy_path
reloading plugin Default.delete_word
reloading plugin Default.detect_indentation
reloading plugin Default.duplicate_line
reloading plugin Default.echo
reloading plugin Default.exec
reloading plugin Default.fold
reloading plugin Default.font
reloading plugin Default.goto_line
reloading plugin Default.history_list
reloading plugin Default.indentation
reloading plugin Default.kill_ring
reloading plugin Default.mark
reloading plugin Default.new_templates
reloading plugin Default.open_file_settings
reloading plugin Default.open_in_browser
reloading plugin Default.pane
reloading plugin Default.paragraph
reloading plugin Default.paste_from_history
reloading plugin Default.save_on_focus_lost
reloading plugin Default.scroll
reloading plugin Default.set_unsaved_view_name
reloading plugin Default.side_bar
reloading plugin Default.sort
reloading plugin Default.swap_line
reloading plugin Default.switch_file
reloading plugin Default.symbol
reloading plugin Default.transform
reloading plugin Default.transpose
reloading plugin Default.trim_trailing_white_space
reloading plugin CSS.css_completions
reloading plugin Diff.diff
reloading plugin HTML.encode_html_entities
reloading plugin HTML.html_completions
reloading plugin Package Control.Package Control
plugins loaded
loaded 837 snippets
Writing file /D/Work/freelancer/es tu ganga/html/css/estuganga3.css with encoding UTF-8 (atomic)
Package Control: No updated packages

I'm using ST3 and installed the package earlier using the instructions given in the comment by Rafal.

**Comment by sergeche** [comment]
I don’t LiveStyle package in list. Did you installed it? Or just Package Control, as described by Rafal?

**Comment by turdi** [comment]
ok, sorry!!!! Now it works!!! I appologize for the mistake!

**Comment by hotspring** [comment]
Thanks for tony1157 's reminded, livestyle's websocket will connect the localhost:54000, but when i modify the hosts to some other IP(not 127.0.0.1),it will can not connect to the ST.
and u will get the "Emmet LiveStyle: Cannot send message, client list empty" log. 
I just  modify back the hosts and then solved the "no active editor" issue :)

**Comment by anthonyartese** [comment]
I'm having this issue. What was the determined fix? Do I have to change a port?

Socket error: 
Event {clipboardData: undefined, cancelBubble: false, returnValue: true, srcElement: WebSocket, defaultPrevented: false…}

ST console:
startup, version: 2221 windows x64 channel: stable
executable: /C/Program Files/Sublime Text 2/sublime_text.exe
working dir: /C/ProgramData/Microsoft/Windows/Start Menu/Programs
packages path: /C/Users/221031421/AppData/Roaming/Sublime Text 2/Packages
settings path: /C/Users/221031421/AppData/Roaming/Sublime Text 2/Settings
PackageSetup not required
Py_GetProgramName(): sublime_text.exe
Py_GetExecPrefix(): C:\Program Files\Sublime Text 2
Py_GetProgramFullPath(): C:\Program Files\Sublime Text 2\sublime_text.exe
Py_GetPath(): C:\Program Files\Sublime Text 2\python26.zip;C:\Program Files\Sublime Text 2\DLLs;C:\Program Files\Sublime Text 2\lib;C:\Program Files\Sublime Text 2\lib\plat-win;C:\Program Files\Sublime Text 2\lib\lib-tk;C:\Program Files\Sublime Text 2
Py_GetPythonHome(): C:\Program Files\Sublime Text 2
catalogue loaded
found 4 files for base name Default.sublime-keymap
found 1 files for base name Default.sublime-mousemap
found 3 files for base name Main.sublime-menu
Reloading plugin C:\Users\221031421\AppData\Roaming\Sublime Text 2\Packages\CSS\css_completions.py
Reloading plugin C:\Users\221031421\AppData\Roaming\Sublime Text 2\Packages\Default\comment.py
Reloading plugin C:\Users\221031421\AppData\Roaming\Sublime Text 2\Packages\Default\copy_path.py
Reloading plugin C:\Users\221031421\AppData\Roaming\Sublime Text 2\Packages\Default\delete_word.py
Reloading plugin C:\Users\221031421\AppData\Roaming\Sublime Text 2\Packages\Default\detect_indentation.py
Reloading plugin C:\Users\221031421\AppData\Roaming\Sublime Text 2\Packages\Default\duplicate_line.py
Reloading plugin C:\Users\221031421\AppData\Roaming\Sublime Text 2\Packages\Default\echo.py
Reloading plugin C:\Users\221031421\AppData\Roaming\Sublime Text 2\Packages\Default\exec.py
Reloading plugin C:\Users\221031421\AppData\Roaming\Sublime Text 2\Packages\Default\fold.py
Reloading plugin C:\Users\221031421\AppData\Roaming\Sublime Text 2\Packages\Default\font.py
Reloading plugin C:\Users\221031421\AppData\Roaming\Sublime Text 2\Packages\Default\goto_line.py
Reloading plugin C:\Users\221031421\AppData\Roaming\Sublime Text 2\Packages\Default\indentation.py
Reloading plugin C:\Users\221031421\AppData\Roaming\Sublime Text 2\Packages\Default\kill_ring.py
Reloading plugin C:\Users\221031421\AppData\Roaming\Sublime Text 2\Packages\Default\mark.py
Reloading plugin C:\Users\221031421\AppData\Roaming\Sublime Text 2\Packages\Default\new_templates.py
Reloading plugin C:\Users\221031421\AppData\Roaming\Sublime Text 2\Packages\Default\open_file_settings.py
Reloading plugin C:\Users\221031421\AppData\Roaming\Sublime Text 2\Packages\Default\open_in_browser.py
Reloading plugin C:\Users\221031421\AppData\Roaming\Sublime Text 2\Packages\Default\paragraph.py
Reloading plugin C:\Users\221031421\AppData\Roaming\Sublime Text 2\Packages\Default\save_on_focus_lost.py
Reloading plugin C:\Users\221031421\AppData\Roaming\Sublime Text 2\Packages\Default\scroll.py
Reloading plugin C:\Users\221031421\AppData\Roaming\Sublime Text 2\Packages\Default\set_unsaved_view_name.py
Reloading plugin C:\Users\221031421\AppData\Roaming\Sublime Text 2\Packages\Default\side_bar.py
Reloading plugin C:\Users\221031421\AppData\Roaming\Sublime Text 2\Packages\Default\sort.py
Reloading plugin C:\Users\221031421\AppData\Roaming\Sublime Text 2\Packages\Default\swap_line.py
Reloading plugin C:\Users\221031421\AppData\Roaming\Sublime Text 2\Packages\Default\switch_file.py
Reloading plugin C:\Users\221031421\AppData\Roaming\Sublime Text 2\Packages\Default\transform.py
Reloading plugin C:\Users\221031421\AppData\Roaming\Sublime Text 2\Packages\Default\transpose.py
Reloading plugin C:\Users\221031421\AppData\Roaming\Sublime Text 2\Packages\Default\trim_trailing_white_space.py
Reloading plugin C:\Users\221031421\AppData\Roaming\Sublime Text 2\Packages\Diff\diff.py
Reloading plugin C:\Users\221031421\AppData\Roaming\Sublime Text 2\Packages\HTML\encode_html_entities.py
Reloading plugin C:\Users\221031421\AppData\Roaming\Sublime Text 2\Packages\HTML\html_completions.py
Reloading plugin C:\Users\221031421\AppData\Roaming\Sublime Text 2\Packages\LiveStyle\livestyle.py
Reloading plugin C:\Users\221031421\AppData\Roaming\Sublime Text 2\Packages\Package Control\Package Control.py
plugin init time: 0.607234
loading bindings
loading pointer bindings
found 1 files for base name Default.sublime-theme
theme loaded
app ready
pre session restore time: 0.924225
wrote startup cache, added files: 4 orphaned files: 0 total files: 98 cache hits: 94
startup time: 1.0283 (package setup was not run)
loaded 837 snippets
Package Control: Skipping automatic upgrade, last run at 2013-09-10 12:37:43, next run at 2013-09-10 13:37:43 or after

**Comment by sergeche** [comment]
Hi,  

try to restart both Sublime Text and Chrome. Also, try to enable debug mode for ST plugin: https://github.com/emmetio/livestyle-sublime/blob/master/messages/install.txt
It will give you more insights in console about what’s going on

---

Сергей Чикуёнок
http://chikuyonok.ru
serge.che@gmail.com

**Comment by anthonyartese** [comment]
with debug mode enabled im seeing 'Emmet LiveStyle: Cannot send message, client list empty'. Not sure what that means. Here is the full log: 

startup, version: 2221 windows x64 channel: stable
executable: /C/Program Files/Sublime Text 2/sublime_text.exe
working dir: /C/ProgramData/Microsoft/Windows/Start Menu/Programs
packages path: /C/Users/221031421/AppData/Roaming/Sublime Text 2/Packages
settings path: /C/Users/221031421/AppData/Roaming/Sublime Text 2/Settings
PackageSetup not required
Py_GetProgramName(): sublime_text.exe
Py_GetExecPrefix(): C:\Program Files\Sublime Text 2
Py_GetProgramFullPath(): C:\Program Files\Sublime Text 2\sublime_text.exe
Py_GetPath(): C:\Program Files\Sublime Text 2\python26.zip;C:\Program Files\Sublime Text 2\DLLs;C:\Program Files\Sublime Text 2\lib;C:\Program Files\Sublime Text 2\lib\plat-win;C:\Program Files\Sublime Text 2\lib\lib-tk;C:\Program Files\Sublime Text 2
Py_GetPythonHome(): C:\Program Files\Sublime Text 2
catalogue loaded
found 4 files for base name Default.sublime-keymap
found 1 files for base name Default.sublime-mousemap
found 3 files for base name Main.sublime-menu
Reloading plugin C:\Users\221031421\AppData\Roaming\Sublime Text 2\Packages\CSS\css_completions.py
Reloading plugin C:\Users\221031421\AppData\Roaming\Sublime Text 2\Packages\Default\comment.py
Reloading plugin C:\Users\221031421\AppData\Roaming\Sublime Text 2\Packages\Default\copy_path.py
Reloading plugin C:\Users\221031421\AppData\Roaming\Sublime Text 2\Packages\Default\delete_word.py
Reloading plugin C:\Users\221031421\AppData\Roaming\Sublime Text 2\Packages\Default\detect_indentation.py
Reloading plugin C:\Users\221031421\AppData\Roaming\Sublime Text 2\Packages\Default\duplicate_line.py
Reloading plugin C:\Users\221031421\AppData\Roaming\Sublime Text 2\Packages\Default\echo.py
Reloading plugin C:\Users\221031421\AppData\Roaming\Sublime Text 2\Packages\Default\exec.py
Reloading plugin C:\Users\221031421\AppData\Roaming\Sublime Text 2\Packages\Default\fold.py
Reloading plugin C:\Users\221031421\AppData\Roaming\Sublime Text 2\Packages\Default\font.py
Reloading plugin C:\Users\221031421\AppData\Roaming\Sublime Text 2\Packages\Default\goto_line.py
Reloading plugin C:\Users\221031421\AppData\Roaming\Sublime Text 2\Packages\Default\indentation.py
Reloading plugin C:\Users\221031421\AppData\Roaming\Sublime Text 2\Packages\Default\kill_ring.py
Reloading plugin C:\Users\221031421\AppData\Roaming\Sublime Text 2\Packages\Default\mark.py
Reloading plugin C:\Users\221031421\AppData\Roaming\Sublime Text 2\Packages\Default\new_templates.py
Reloading plugin C:\Users\221031421\AppData\Roaming\Sublime Text 2\Packages\Default\open_file_settings.py
Reloading plugin C:\Users\221031421\AppData\Roaming\Sublime Text 2\Packages\Default\open_in_browser.py
Reloading plugin C:\Users\221031421\AppData\Roaming\Sublime Text 2\Packages\Default\paragraph.py
Reloading plugin C:\Users\221031421\AppData\Roaming\Sublime Text 2\Packages\Default\save_on_focus_lost.py
Reloading plugin C:\Users\221031421\AppData\Roaming\Sublime Text 2\Packages\Default\scroll.py
Reloading plugin C:\Users\221031421\AppData\Roaming\Sublime Text 2\Packages\Default\set_unsaved_view_name.py
Reloading plugin C:\Users\221031421\AppData\Roaming\Sublime Text 2\Packages\Default\side_bar.py
Reloading plugin C:\Users\221031421\AppData\Roaming\Sublime Text 2\Packages\Default\sort.py
Reloading plugin C:\Users\221031421\AppData\Roaming\Sublime Text 2\Packages\Default\swap_line.py
Reloading plugin C:\Users\221031421\AppData\Roaming\Sublime Text 2\Packages\Default\switch_file.py
Reloading plugin C:\Users\221031421\AppData\Roaming\Sublime Text 2\Packages\Default\transform.py
Reloading plugin C:\Users\221031421\AppData\Roaming\Sublime Text 2\Packages\Default\transpose.py
Reloading plugin C:\Users\221031421\AppData\Roaming\Sublime Text 2\Packages\Default\trim_trailing_white_space.py
Reloading plugin C:\Users\221031421\AppData\Roaming\Sublime Text 2\Packages\Diff\diff.py
Reloading plugin C:\Users\221031421\AppData\Roaming\Sublime Text 2\Packages\HTML\encode_html_entities.py
Reloading plugin C:\Users\221031421\AppData\Roaming\Sublime Text 2\Packages\HTML\html_completions.py
Reloading plugin C:\Users\221031421\AppData\Roaming\Sublime Text 2\Packages\LiveStyle\livestyle.py
Reloading plugin C:\Users\221031421\AppData\Roaming\Sublime Text 2\Packages\Package Control\Package Control.py
plugin init time: 4.67193
loading bindings
loading pointer bindings
found 1 files for base name Default.sublime-theme
theme loaded
app ready
pre session restore time: 11.4497
Emmet LiveStyle: Prepare diff
startup cache, total files: 98 cache hits: 98
startup time: 11.7286 (package setup was not run)
Emmet LiveStyle: Cannot send message, client list empty
Emmet LiveStyle: Cannot send message, client list empty
Emmet LiveStyle: Cannot send message, client list empty
Package Control: No updated packages
loaded 837 snippets

**Comment by sergeche** [comment]
@anthonyartese it means there’s no connected browser. Do you use a proxy or something?

**Comment by anthonyartese** [comment]
I'm on my work network. Not sure what they have set up. What would I have to do to troubleshoot that?

**Comment by socialblogsite** [comment]
My computer has manual DNS, but AFAIK 127.0.0.1 is still there. 

I just closed MAMP, cleared the DNS cache, nothing. Inspecting any page (mamp or not) shows the same socket error. Do I have reset my computer to a stock OS for this to work?

I never played with sockets and the hacker thing I did was installing Apache2, or mySQL but after Mavericks Apache is gone and I just got Mamp.

Could "Save CSS" scripts be messing with this? I uninstalled it, deleted it, restarted Chrome, but I STILL see a "save" option in the contextual menu, (and it works) so it seems some script is still working.
